lib/aws-manager.js

```diff
@@ -1300,37 +1151,18 @@ class AwsManager {
     let i = instances || [];
     let r = requests || [];
 
-    if (i.length > 0) {
-      debug('killing instances %s in %j', i, region);
-      try {
-        await this.ec2.terminateInstances.inRegion(region, {
-          InstanceIds: i,
-        });
-      } catch (err) {
-        debug('error terminating instances in %s', region);
-        debug(err);
-        if (err.stack) {
-          debug(err.stack);
-        }
-        throw err;
-      }
-    }
+    let promises = [];
 
-    if (r.length > 0) {
-      debug('cancelling spot requests %j in %s', r, region);
-      try {
-        await this.ec2.cancelSpotInstanceRequests.inRegion(region, {
-          SpotInstanceRequestIds: r,
-        });
-      } catch (err) {
-        debug('error cancelling spot requests in %s', region);
-        debug(err);
-        if (err.stack) {
-          debug(err.stack);
-        }
-        throw err;
-      }
-    }
+    //if (i.length > 0) {
```

Comment:
heh, yes, there was also a bug where I left off the .promise() method... I hate that we need to call the .promise() method... I'd so much rather have x + 'Async' pattern.